### PR TITLE
Digit 2.5 Upgrade related changes

### PR DESCRIPTION
**Pull request by khemanth-eGov**

### CODE REVIEW

**Review by pradeepkumarcm-egov — APPROVED** [review]

**Review by sathishp-eGov — COMMENTED** [review]
@khemanth-eGov and @LataNaik-eGov please check the following things

1. sms-enabled: "false" -- SMS should be enabled in staging.

2. oauth2-proxy:
  config:
    configFile: |-
      email_domains = [ "*" ]
      github_org = "egovernments"
      github_team = "micro-service-uat,devops"
      upstreams = [ "file:///dev/null" ]     -- UAT details present here

**Comment by khemanth-eGov** [comment]
1. sms-enabled: "false" -- SMS should be enabled in staging. - Resolved

2. oauth2-proxy:
config:
configFile: |-
email_domains = [ "*" ]
github_org = "egovernments"
github_team = "micro-service-uat,devops"
upstreams = [ "file:///dev/null" ] -- UAT details present here : Removed(Resolved)

**Comment by khemanth-eGov** [comment]
> @khemanth-eGov and @LataNaik-eGov please check the following things
> 
> 1. sms-enabled: "false" -- SMS should be enabled in staging. - Resolved
> 2. oauth2-proxy:
>    config:
>    configFile: |-
>    email_domains = [ "*" ]
>    github_org = "egovernments"
>    github_team = "micro-service-uat,devops"
>    upstreams = [ "file:///dev/null" ]     -- UAT details present here - Resolved

**Review by sathishp-eGov — APPROVED** [review]

File: deploy-as-code/helm/environments/staging.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ cluster-configs:
         serializers-timezone-in-ist: "true"
         server-tomcat-max-threads: "10"
         server-tomcat-max-connections: "500"
-        sms-enabled: "true"
```
[the remark this answers is not in the document]
Comment (khemanth-eGov) [reply deploy-as-code/helm/environments/staging.yaml:51]: Resolved

File: deploy-as-code/helm/environments/staging.yaml [hunk 2]
```diff
@@ -847,6 +863,15 @@ grafana:
   ingress:
     hostName: minio-staging.digit.org
 
+
+oauth2-proxy:
+  config:
+    configFile: |-
+      email_domains = [ "*" ]
+      github_org = "egovernments"
+      github_team = "micro-service-uat,devops"
```
[the remark this answers is not in the document]
Comment (khemanth-eGov) [reply deploy-as-code/helm/environments/staging.yaml]: Removed the oauth part

**Pull request merged**